client/src/components/utils/appHelper.js

```diff
@@ -14,10 +20,19 @@ const autofillSearchBar = (props, interactionTypes) => {
   return props.destination.name;
 };
 
+/**
+ * Move the center of the map to the origin (no return statement).
+ * @param  {Object} props Props passed into the App component
+ */
 const centerMapToOrigin = (props) => {
   props.map.setCenter(props.position);
 };
 
+/**
+ * Make request to PlaceService API to convert place into Lat/Lng.
+ * @param  {Object} place Origin or destination place information
+ * @return {Object}       Origin or destinatin Lat/Lng
```

Comment:
"Origin or destination **with** Lat/Lng" ?

I feel like how it's currently worded, I would expect it to only return the lat/lng